src/components/presentational/NewsDetail.js

```diff
@@ -96,6 +156,18 @@ export default class NewsDetail extends Component {
               </span>
             }
             {' | '}
+            {
+                <a href="javascript:void(0)" onClick={this.saveStateToLocalStorage}>
+                  {(() => {
+                    switch (this.state.bookmarked) {
+                      case true: return 'unbookmark';
+                      case false: return 'bookmark';
+                      default: return 'bookmark';
```

Comment:
Could have been done with 
`this.state.bookmarked ? 'unboomarked' : 'bookmarked'`